pol-core/pol/miscmsg.cpp

```diff
@@ -467,12 +467,14 @@ void handle_unknown_C4( Client* client, PKTOUT_C4* /*msg*/ )
 void handle_update_range_change( Client* client, PKTBI_C8* msg )
 {
   // limit range to official range 5-24
-  u8 range = std::clamp( msg->range, (u8)5, (u8)24 );
-  client->set_update_range( range );
+  if ( settingsManager.ssopt.allow_visual_range_modification )
+  {
+    u8 range = std::clamp( msg->range, (u8)5, (u8)24 );
```

Comment:
I don't want to be annoying, but could this be a configuration as well - something like `MinimumVisualRange` / `MaximumVisualRange`? If we were to allow clients changing it by themselves on our shard we'd most likely want to allow only to lower it.

Reply:
Added

Reply:
Resolving as this was addressed